ui/app/src/components/Controls/SearchAhead.tsx

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
Let's discuss this control's name, location and approach. 
At the moment, the use is on the AddressBar so on this location and with this name, appears to be reusable but it's not truly generic